package.json

```diff
@@ -8,7 +8,7 @@
     "autoplay-whitelist": "github:brave/autoplay-whitelist",
     "aws-sdk": "^2.449.0",
     "brave-site-specific-scripts": "github:brave/brave-site-specific-scripts",
-    "ethereum-remote-client": "^0.1.81",
+    "ethereum-remote-client": "^0.1.83",
```

Comment:
Additional version bump is due to `1.0.82` failing on account of an OTP provided too late